README.md

```diff
@@ -14,48 +14,35 @@ The [EuropeanFootballLeaguePredictor Github Page](https://nickpadd.github.io/Eur
 
 
 ## How to run the project
-### Installing Project Dependencies
-1. Clone this repository to your local machine or download the project files.
+### Haven’t installed the project yet? Time to dive in!
 
-   ```bash
-   git clone https://github.com/nickpadd/EuropeanFootballLeaguePredictor
-
-2. Navigate to the project directory using the command line.
-    ```bash
-    cd EuropeanFootballLeaguePredictor
-
-- Managing dependencies with Poetry: 
-    - Installing poetry:
-    ```bash
-    pip install poetry 
-    ```
-    - Installing project dependencies:
-    ```bash
-    poetry install
-    ```
+1. Download the file *PROJECT-SETUP.sh* (**or the '.bat' for Windows users**) from the repository and place it in the
+folder where you want to install the project.
+2. Open the terminal and navigate to the folder where the 'PROJECT-SETUP.sh' file is located.
+3. Run the following command in the terminal:
+- macOS/Linux users
+```bash
+bash PROJECT-SETUP.sh
+```
+- Windows users
+```bash
+PROJECT-SETUP.bat
+```
```

Comment:
If the `.sh` and `.bat` scripts are modified to exclude the cloning procedure, these lines should explicitly specify the steps for cloning the repository.